python/paddle/fluid/executor.py

```diff
@@ -2034,8 +2034,11 @@ def _prepare_fleet_executor_carrier(self,
             fleet_opt['task_id_to_rank'] = task_id_to_rank
         place = core.Place()
         place.set_place(self.place)
+        # NOTE: the last argument is used to force create some vars in root scope,
+        # won't be used during train.
         self._fleet_executor.init(carrier_id, program.desc, scope, place,
-                                  num_micro_batches, tasks, task_id_to_rank)
+                                  num_micro_batches, tasks, task_id_to_rank,
```

Comment:
If the last parameter only used in Inference, it should be seen on it's name.

Reply:
done udpated